### DISCUSSION TITLE
[Help Wanted]  Invalid password / Unknown error when trying to login

### DISCUSSION THREAD

**Discussion by tg5hKVHPyX** [standardnotes/forum#3010] [General]
**Describe the issue**
Since September or October of last year, I'm no longer able to log in to my account. It's a free account on the official instance if that matters. This started happening since I wiped my phone which was the only device I was successfully logged in on, and trying to login from the web app gives me the same errors since then. I get an "Invalid password" error and now more recently, I've gotten an "Unknown error" when trying to log in. I'm certain that there's no problem with the password as I use a password manager and haven't changed the password since creating the account. I really need to access this account and back up the notes from it, there's a lot of important stuff on there. I've contacted the help email and they told me there's nothing they can do since accounts can't be restored, but I don't want to give up. Has anyone else had this same problem?

**To Reproduce**
1. Go to sign in.
2. Enter email and password.
3. Click enter.
4. See error.

**Expected behavior**
Successful login to my account, like I used to be able to.

**Smartphone (please complete the following information):**
 - Device: Samsung A70
 - OS: Android 10
 - Browser: Firefox
 - Version: the error started happening since last October, current version is 111.1.1
 - Also using the latest version of the app that's released on f-droid which is 3.150.37

**Comment by Peltiloota** [comment]
There's sadly very little anyone can do without the correct password. Is it possible that you used another email address, if you're certain about the password being correct? 
https://standardnotes.com/help/6/i-ve-forgotten-my-password-what-should-i-do

**Comment by tg5hKVHPyX** [comment]
> There's sadly very little anyone can do without the correct password. Is it possible that you used another email address, if you're certain about the password being correct? https://standardnotes.com/help/6/i-ve-forgotten-my-password-what-should-i-do

When I reached out to help@standardnotes.com, the person replying confirmed that an account with my email address does exist. So I am sure that both the email and password are correct.
